openapi/atlas-api-transformed.yaml

```diff
@@ -21833,11 +21833,6 @@ components:
           type: string
           description: Application error code returned with this error.
           example: TOO_MANY_GROUP_NOTIFICATIONS
-        parameters:
-          type: array
-          items:
-            type: object
```

Comment:
reading the ticket I guess the problem comes because we define this an as array of object, but what we mean is an array of "anything", being an array of object openapi-generator is assuming `[]map[string]interface{}` but it should be `[]interface{}` what's the best way to model this on openapi?

Reply:
My line of thinking - what is the value of that in error. 
We are returning parameters that user sent?

Reply:
Ok. Ignore me